src/main/java/org/commcare/formplayer/parsers/FormplayerCaseXmlParser.java

```diff
@@ -16,6 +16,8 @@ public class FormplayerCaseXmlParser extends CaseXmlParser {
     private final EntityStorageCache mEntityCache;
     private final FormplayerCaseIndexTable mCaseIndexTable;
 
+    public static final String CASE_INDEX_STORAGE_TABLE_NAME = "case_index_storage";
```

Comment:
Why is this defined here? What about moving it to `FormplayerCaseIndexTable` where it's actually used?

Reply:
https://github.com/dimagi/formplayer/pull/1351/commits/26a1b9d43cd8812c547880590dd9ed6776d95558